### PR TITLE
refactor: helper_functions.py

### PR DESCRIPTION
**Pull request by zt4ff**

### CODE REVIEW

File: src/utils/helper/detect_icon.py
Comment (maximiliangarmatsch) [inline src/utils/helper/detect_icon.py:1]: move please...
e.g "/components/pyautogui/detect_icon.py"

File: src/utils/helper/extract_pdf_text.py
Comment (maximiliangarmatsch) [inline src/utils/helper/extract_pdf_text.py:1]: move please...
e.g "/components/pdfminer/extract_pdf_text.py"

File: src/utils/helper/is_osx.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+"""
+helper funtions pyautogui_gmail.py
```
Comment (maximiliangarmatsch) [inline src/utils/helper/is_osx.py:2]: no need for comment

File: src/utils/helper_funtion.py [hunk 2]
```diff
@@ -199,7 +157,7 @@ def process_unread_emails(browser, unread_emails):
 {attachment_email_response}
 """
```
Comment (maximiliangarmatsch) [inline src/utils/helper_funtion.py:158]: comment needed?

File: src/utils/pyautogui_gmail.py
Comment (maximiliangarmatsch) [inline src/utils/pyautogui_gmail.py:1]: move please...
e.g "/components/pyautogui/gmail.py"

File: src/utils/helper_funtion.py (hunk 2 above)
Comment (zt4ff) [reply src/utils/helper_funtion.py:158]: I think so, I'd ask Amir.

**Review by maximiliangarmatsch — APPROVED** [review]

**Pull request merged**